units/src/amount/tests.rs

```diff
@@ -96,17 +96,22 @@ fn from_str_zero_without_denomination() {
 
 #[test]
 fn from_int_btc() {
+    let sat = Amount::from_sat;
```

Comment:
Good idea, we can later just change this to `|x| Amount::from_sat(x).unwrap`.
But it's repeated in multiple tests so why not just go all the way and define the function only once for the whole module? We can then just change one definition and it'll work fine.

Reply:
No worries, I can do that.